group_vars/matrix_servers

```diff
@@ -4524,6 +4526,37 @@ matrix_client_schildichat_jitsi_preferred_domain: "{{ matrix_server_fqn_jitsi if
 #
 ######################################################################
 
+######################################################################
+#
+# matrix-client-fluffychat
+#
+######################################################################
+
+matrix_client_fluffychat_enabled: false
+
+matrix_client_fluffychat_scheme: "{{ 'https' if matrix_playbook_ssl_enabled else 'http' }}"
+
+matrix_client_fluffychat_container_image_self_build: "{{ matrix_architecture not in ['arm64', 'amd64'] }}"
+
+matrix_client_fluffychat_container_http_host_bind_port: "{{ (matrix_playbook_service_host_bind_interface_prefix ~ '8765') if matrix_playbook_service_host_bind_interface_prefix else '' }}"
```

Comment:
```suggestion
matrix_client_fluffychat_container_http_host_bind_port: "{{ (matrix_playbook_service_host_bind_interface_prefix ~ '8770') if matrix_playbook_service_host_bind_interface_prefix else '' }}"
```

`8765` is taken by element-web, and so are the other ports after it.

`8770` is currently the first free one.